incubator/scylladb/templates/statefulset.yaml

```diff
@@ -0,0 +1,97 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ template "scylladb.fullname" . }}
+  labels:
+    app: {{ template "scylladb.name" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ template "scylladb.name" . }}
+  serviceName: {{ template "scylladb.fullname" . }}
+  replicas: {{ .Values.replicaCount }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "scylladb.name" . }}
+        release: {{ .Release.Name }}
+    spec:
+      terminationGracePeriodSeconds: 120
+      containers:
+      - name: {{ template "scylladb.fullname" . }}
+        image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
+        resources: 
+{{ toYaml .Values.resources | indent 10 }}
+        ports:
+          - containerPort: {{ .Values.service.cqlPort }}
+          - containerPort: {{ .Values.service.internodePort }}
+          - containerPort: {{ .Values.service.sslinternodePort }}
+          - containerPort: {{ .Values.service.jmxPort }}
+          - containerPort: {{ .Values.service.restPort }}
+          - containerPort: {{ .Values.service.prometheusPort }}
+          - containerPort: {{ .Values.service.nodeExporterPort }}
+          - containerPort: {{ .Values.service.thriftPort }}
+        env:
+        - name: MY_POD_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.name
+        - name: MY_SERVICE_NAME
+          value: {{ template "scylladb.fullname" . }}
+        - name: POD_IP
+          valueFrom:
+            fieldRef:
+              fieldPath: status.podIP
+        args:
+          - --seeds 
+          - {{ template "scylladb.fullname" . }}-0.{{ template "scylladb.fullname" . }}.default.svc.cluster.local
```

Comment:
should be `{{ template "scylladb.fullname" . }}-0.{{ template "scylladb.fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local`